tests/model/test_exif.py

```diff
@@ -27,17 +29,19 @@ def test_tiff(self):
         self.assertEqual(exif.photometricInterpretation, 'RGB')
 
     def test_png1(self):
-        exif = OcrdExif(Image.open(assets.path_to('kant_aufklaerung_1784-binarized/data/OCR-D-IMG-BIN/BIN_0020.png')))
+        with Image.open(assets.path_to('kant_aufklaerung_1784-binarized/data/OCR-D-IMG-BIN/BIN_0020.png')) as img:
+            exif = OcrdExif(img)
         self.assertEqual(exif.width, 1457)
         self.assertEqual(exif.height, 2084)
-        self.assertEqual(exif.xResolution, 300)
-        self.assertEqual(exif.yResolution, 300)
-        self.assertEqual(exif.resolution, 300)
+        self.assertEqual(exif.xResolution, 295)
+        self.assertEqual(exif.yResolution, 295)
+        self.assertEqual(exif.resolution, 295)
```

Comment:
```suggestion
        self.assertEqual(exif.resolution, 295)
        self.assertEqual(exif.resolutionUnit, 'inches')
```

Reply:
(And maybe the check for the unit attribute can also be extended to the other test cases.)